website/docs/components/logo.mdx

```diff
@@ -0,0 +1,11 @@
+import { StorybookComponent } from "@site/src/components/StorybookComponent";
+
```

Comment:
Het Logo component komt nu met kleine letters in het menu:

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/aac39821-8718-43c3-9d14-98128d417438)

Dat komt omdat deze mdx file geen heading heeft:

```
# Logo
```

Zie bv. header.mdx.